src/responder/gasQueue.ts

```diff
@@ -285,4 +286,54 @@ export class GasQueue {
         clonedArray.shift();
         return new GasQueue(clonedArray, this.emptyNonce, this.replacementRate, this.maxQueueDepth);
     }
+
+    /**
+     * Returns all queue items that are in this queue but not in the supplied queue
+     * @param otherQueue
+     */
+    public difference(otherQueue: GasQueue): GasQueueItem[] {
+        return this.queueItems.filter(tx => !otherQueue.queueItems.includes(tx));
+    }
+
```

Comment:
This has quadratic complexity, which could be fixed easily by temporarily making a Set for the otherQueue items.

Reply:
I'm not sure it matters too much, since gas queues should be small. But it's an quick change so I've added it.

Reply:
Agreed, but you never know where code ends up copy-pasted, so I prefer to fix these things, or annotate the inefficiency.